CHANGELOG.md

```diff
@@ -1,5 +1,109 @@
 # Changelog
 
+## 0.61.0
+
+[TODO: Description]
+
+### Added
+
+* Add Fast Refresh by default ([17f8e58](https://github.com/facebook/react-native/commit/17f8e58) by [@gaearon](https://github.com/gaearon))
+* Add `useWindowDimensions` hook to replace most `Dimensions` usage ([5ec382d](https://github.com/facebook/react-native/commit/5ec382d) by [@sahrens](https://github.com/sahrens))
+* Add Flipper support ([3a66fc7](https://github.com/facebook/react-native/commit/3a66fc7) by [@axe-fb](https://github.com/axe-fb))
+* Add explicit `useNativeDriver: false` to callsites ([0a68763](https://github.com/facebook/react-native/commit/0a68763) by [@TheSavior](https://github.com/TheSavior))
```

Comment:
I'm not sure if we should mention this. There is no deprecation warning today, just a change in the flow type. Technically that is breaking change to our API but so few people use Flow that I'm not sure if is a big deal. 

We want to make this change but just haven't so far. Before including it in a changelog we should probably also have a deprecation warning saying it is required, and also update the TypeScript definition. I had previously talked with @acoates-ms about the TypeScript change for this.

Reply:
If we do include this, the changelog line should probably be something like this instead, since it isn't an addition.

```
Require `useNativeDriver` to be explicitly set to true or false
```

Reply:
Is there a runtime warning / yellowbox if you don't explicitly set it?

Reply:
Not currently. I'd accept a PR that added one though! :)